### ISSUE TITLE
It won't print all my citations.

### ISSUE THREAD

**Issue by VasTsak** [joaomlourenco/novathesis#64]
Hello, I have written the report in latex (my own script) and then they told me that I should adjust it to this template. In my script it will print out all the citations, while in this template it will only print out a small portion.  

This is one example of the citations that won't print out : 

@article{Pennington2014,
author = {Pennington, Jeffrey and Socher, Richard and Manning, Christopher D},
file = {:home/vasilis/Desktop/D14-1162 (2).pdf:pdf},
mendeley-groups = {Weight initialitation},
pages = {1532--1543},
title = {{GloVe : Global Vectors for Word Representation}},
year = {2014}
}

Is it a problem in the structure of that? 

By the way, I have exported the bib citation by Mendley.

**Comment by joaomlourenco** [comment]
Hi,
Mendeley leaves lots and lots and lots of "garbage" in the exported bib file.
Please process your bibliography in the command line using
"bibtex template"
and start cleaning up all entries that bibtex complains.
In the end it shall work.
João L.

**Comment by VasTsak** [comment]
Hello Joao, 

Thanks for the immediate response, after all it was not the problem. But I fixed it, and I am assuming that at some point someone else will have the same problem. So below is the way I fixed the problem. 

1. https://tex.stackexchange.com/questions/86482/no-bibliography-is-printed-and-citations-shown-bibtex-key 

The second link is referred to those who use 'texmaker' to compile Latex. 

2. https://tex.stackexchange.com/questions/44040/biblatex-biber-texmaker-miktex
